packages/tracking/composer.json

```diff
@@ -5,6 +5,7 @@
 	"license": "GPL-2.0-or-later",
 	"require": {
 		"automattic/jetpack-options": "@dev",
+		"automattic/jetpack-status": "@dev",
 		"automattic/jetpack-terms-of-service": "@dev"
```

Comment:
Since we're not using `Terms_Of_Service` in the package anymore, should we get rid of that dependency?

Reply:
We do use it when creating the instance to pass into the new method [here](https://github.com/Automattic/jetpack/blob/master/packages/tracking/src/class-tracking.php#L103). Thanks for being mindful of the dependencies, it's good to have a sanity check on those every once in a while!